packages/common/src/DateUtils.ts

```diff
@@ -102,6 +102,25 @@ export function DateWithAddedMinutes(date: Date, minutes: number) {
   return new Date(date.getTime() + minutes * 60000);
 }
 
+export function getNumberOfDaysBetweenDates(
+  start: Date | number,
+  end: Date | number
+) {
+  const date1 = new Date(start);
+  const date2 = new Date(end);
+
+  // One day in milliseconds
+  const oneDay = 1000 * 60 * 60 * 24;
+
+  // Calculating the time difference between two dates
+  const diffInTime = date1.getTime() - date2.getTime();
+
+  // Calculating the no. of days between two dates
+  const diffInDays = Math.round(diffInTime / oneDay);
```

Comment:
Should we use `Math.ceil()`?
What would `ceil` or `round` return when the difference is a bit less than a full day? What is the expected behaviour?

Reply:
Actually copied this sample from the web but 
`round` is the right solution here, `ceil` rounds to the closest number (5.7 -> 6, 5.3 -> 5) while `round` rounds to the lower value (5.7 -> 5, 5.3 -> 5). Since we need the number of full days round is the right one

Reply:
Awesome!
BTW, the fact we had this conversation here tells me there aren't enough tests for this function.
I suggest you add 2 more functions, one that checks a case where the diff is a bit more than a (number of) full days, and one that is a bit less.